### PR TITLE
Harvest in force mode when settings modified

### LINKED ISSUE #593: Detect changes in processing parameters and force processing
If `thinkhazard_processing.yaml` is changed this means that the processing may need to be done again.
Forcing harvesting again should do the job.

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Fix #593

### CODE REVIEW

**Comment by pgiraud** [comment]
To be rebased and down_revision to be changed.

File: README.rst [hunk 1]
```diff
@@ -218,6 +218,9 @@ Configuration  of processing parameters
 The configuration of the threshold, return periods and units for the different
 hazard types can be done via the `thinkhazard_processing.yaml`.
 
+After any modification to this file, next harvesting will delete all layers,
+hazardsets and processing outputs. This meens that next processing task will
```
Comment (pgiraud) [inline README.rst]: means

File: alembic/versions/06bbf407cad3_add_table_processing_harvesting.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+down_revision = 'd37717f1163b'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    ### commands auto generated by Alembic - please adjust! ###
```
Comment (pgiraud) [inline alembic/versions/06bbf407cad3_add_table_processing_harvesting.py]: We can remove the alembic autogenerated comments.

**Comment by arnaud-morvan** [comment]
Done

**Comment by pgiraud** [comment]
LGTM. Please merge.

In the future, we could enhance this by managing a single table where we keep track of the history of all the possible events (harvesting, complete, processing, and publish).

**Pull request merged**